scss/index.scss

```diff
@@ -30,3 +31,41 @@
   color: $gray;
   padding-top: 5px;
 }
+
+body::-webkit-scrollbar {
+  width: 0.25rem;
+}
+
+body::-webkit-scrollbar-track {
+  background: $white;
+}
+
+body::-webkit-scrollbar-thumb {
+  background: $white;
+}
+
+.navbar {
+  background-color: $white; 
+  box-shadow: 4px 4px 10px 0 $black-lighter-5;
```

Comment:
the shadow is unlikely that be the same. I believe that is a little different. 🤔